content/blog/2019-06-12-taking-over-maint.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2019-06-12
+slug: taking-over-maint
+title: Taking over maintainence of a software package
+authors:
+  - Scott Chamberlain
+  - Maëlle Salmon
+  - Noam Ross
+categories: blog
+topicid: 
+tags:
+  - software
+  - community
+---
+
+Software is maintained by people. While software can in theory live on indefinitely, to do so requires people. People change jobs, move locations, retire, and unfortuntaley die sometimes. When a software maintainer can no longer maintain a package, what happens to the software?
+
+Because of the fragility of people in software, in an ideal world a piece of software should have as many maintainers as possible. Increasing maintainers increases the so-called [bus factor][bus]. A piece of software should have a high of a number as possible; lower means fewer people have to get hit by a bus to then have no maintainers. 
+
+In theory, finding maintainers for software is easier as the pool of users gets larger. However, at rOpenSci, we do generally have this luxury. Most rOpenSci packages have a rather focused and small task, and the pool of users is relatively small. Thus, the average rOpenSci package is maintained by one person, making that package sensitive to the maintainer leaving.
+
+### Small wins
+
+Despite a small number of maintainers per package, the rOpenSci community is quite large. That is, whereas individual R packages that are very popular may have a large enough community on their own, individual packages within rOpenSci can benefit from the large community we've cultivated. 
+
+We've had success finding maintainers by reaching out to our community as maintainers have moved on - we've found new maintainers for all five packages that have needed new maintainers. See our latest [rOpenSci News](https://news.ropensci.org/2019-05-13/) "Call For Contributors" section for recent examples.
+
+### Taking over
+
+Okay, so you've taken over maintainence of a software package. What next? We want to add a section to our [Software Development Guide][devg] about taking over maintainence; we're first reaching out here to the community to talk through what we're thinking and to get your feedback. This is what we're thinking thus far:
+
+#### Proposed material for dev guide
```

Comment:
put empty lines between items, because it'll make them display in a nicer way (their display when there's no empty lines is a design bug)

Reply:
ok